### DISCUSSION TITLE
Timeout during handshake while connecting

### DISCUSSION THREAD

**Discussion by dkadjo** [dask/distributed#5493] [General] [answered]
Hello, i'm trying to Connect to distributed dask cluster like that :
client = Client("tcp://scheduler:8786")
client

And i have this error message 
![WhatsApp Image 2021-11-04 at 13 59 55](https://user-images.githubusercontent.com/37436738/140325937-c207afe6-2f23-49c1-b4de-01192368dc1a.jpeg)
:

How can i fix this error please ?

**Comment by jrbourbeau** [comment, marked as answer]
Hi @dkadjo, thanks for opening this issue! It seems like this question is more of a usage question than a bug report or feature request. We encourage people with such questions to ask them at the [Dask Discourse](https://dask.discourse.group/t/how-to-use-the-dask-discourse/21). Would you mind opening a discussion topic about this over there? If you do so, feel free to close this issue and we will continue the conversation on discourse.

**Reply by dkadjo** [reply]
Thanks @jrbourbeau
